.github/workflows/shared.yml

```diff
@@ -57,12 +56,11 @@ jobs:
   readme-snippets:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v4
+      - uses: actions/checkout@v5
 
-      - uses: astral-sh/setup-uv@v5
+      - uses: astral-sh/setup-uv@v6
         with:
           enable-cache: true
-          version: 0.7.2
```

Comment:
Please put this back.

Reply:
https://github.com/astral-sh/uv/releases/tag/0.8.11 Added Python 3.14rc2 and 0.8.12 speed up zstandard performance.

Reply:
You can change the version, but there's no need to remove it.

We actually need to set this version in the pyproject.toml.